tools/contractdeployer/contract_deployer.go

```diff
@@ -44,10 +44,11 @@ type contractDeployer struct {
 	contractCode []byte
 }
 
-func Deploy(config *Config) error {
+// Deploy deploys the contract specified in the config, and returns its address.
+func Deploy(config *Config) (string, error) {
```

Comment:
We return the address of the deployed contract, for use in the tests (e.g. so we can go check the contract exists).